data_structures/src/chain.rs

```diff
@@ -237,7 +275,57 @@ pub struct Transaction {
     pub signatures: Vec<KeyedSignature>,
 }
 
+/// The error type for operations on a [`Transaction`](Transaction)
+#[derive(Debug, PartialEq)]
+pub enum TransactionError {
+    /// Error indicating the transaction creates value
```

Comment:
I'd rather use this style of error description:
```suggestion
    /// The transaction creates value
```